src/screenConstants.ts

```diff
@@ -1,12 +1,19 @@
-import { ScreenConstantsInterface } from './interfaces'
+import { TaskConstantsInterface, NetworkConstantsInterface } from './interfaces'
+import { colorCodes } from './constants';
 
-export const screenConstants : ScreenConstantsInterface = {
-    'View chain addresses':'addresses',
-    'Check on-chain balance':"balance",
-    'Get network info':"network",
-    'Get validator info':"validators",
-    'Move assets from C-chain to P-chain':'exportCP',
-    'Move assets from P-chain to C-chain':"exportPC",
-    'Add a validator node':"stake",
-    'Delegate to a validator node':"delegate"
-}
\ No newline at end of file
+export const taskConstants: TaskConstantsInterface = {
```

Comment:
you should add the little description for this const as well so that other can get the idea. what and where it can be used.